lib/state/appStore.ts

```diff
@@ -99,7 +99,15 @@ export const useAppStore = create<AppState>()((set, get) => {
   const removeCurrentWallet = () => {
     const wallets = [...get().wallets];
     if (wallets.length <= 1) {
-      // cannot delete last wallet
+      // set to initial wallet status
```

Comment:
I am not sure this check is needed or if we can actually just delete the whole `if`?

Reply:
Wouldn't we need this check? We only do the below if it is the last wallet right?

Reply:
At the bottom is still does:
```
    get().setSelectedWalletId(0);
    set({
      wallets: wallets.filter((_, i) => i !== selectedWalletId),
    });
```

But I think there is actually a bug in the for loop - it will always not work if you try delete the wallet at the last index